src/splitInteger.test.js

```diff
@@ -4,18 +4,31 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const goals = splitInteger(17, 4);
+  const equalParts = goals.slice(0, 3);
 
+  expect(equalParts).toEqual([4, 4, 4]);
```

Comment:
The test checks only the first 3 elements of the array, but it should check the whole array. Use `expect(goals).toEqual([4, 4, 4, 5]);` instead.